packages/edit-post/src/components/header/template-title/style.scss

```diff
@@ -49,10 +49,7 @@
 .edit-post-template-top-area__popover {
 	.components-popover__content {
 		min-width: 280px;
-
-		> div {
-			padding: 0;
```

Comment:
This wasn't affecting the popover so it appeared to be broken (shared screenshots in testing steps).

Reply:
Thank you for sharing!